rasa_sdk/forms.py

```diff
@@ -817,7 +817,12 @@ def slots_mapped_in_domain(self, domain: "DomainDict") -> List[Text]:
         Returns:
             Slot names mapped in the domain.
         """
-        return list(domain.get("forms", {}).get(self.form_name(), {}).keys())
+        return list(
+            domain.get("forms", {})
+            .get(self.form_name(), {})
+            .get("required_slots", {})
```

Comment:
can we please still try to retrieve the slots from the the old format in case somebody uses the new SDK version with an old Rasa Open Source version? 🙌🏻

Reply:
You are so right! 👌🏾 I thought that since we add it in the `domain` in rasa-oss that it will always be there but I did not think about the older versions of Rasa. 🤦🏽‍♀️
I adapted the code and I also tested it by removing (temporarily) the `required_slots` keyword from a test. So it works with / without the `required_slots` keyword now.